packages/cli/test/integration.js

```diff
@@ -307,17 +314,27 @@ async function setupProject(process, projectName, overrides) {
     await waitForPrompt(process, chunk =>
       chunk.includes(`What's your Build Command?`)
     );
-    process.stdin.write(`${buildCommand ?? ''}\n`);
+    process.stdin.write(`${buildCommand || ''}\n`);
 
     await waitForPrompt(process, chunk =>
-      chunk.includes(`What's your Output Directory?`)
+      chunk.includes(`What's your Ignore Command?`)
     );
-    process.stdin.write(`${outputDirectory ?? ''}\n`);
+    process.stdin.write(`${ignoreCommand || ''}\n`);
```

Comment:
I don't think this is correct. We shouldn't as for an ignore command because this isn't a Git deployment, its from the CLI.

Similarly, I don't think we should ask for Install Command since we don't do that on vercel.com/new

Reply:
I'm good with removing the ignoreCommand prompt. But I think having installCommand would be useful. Is there a specific reason when it is not included on `/new`? Or is it just cause it was never added?

Reply:
As we discussed in the meeting, lets save that feature for another PR and keep this PR to the scope of overrides

Reply:
(this question was answered during a call - posting answer here for consistency)

- decided to remove install and ignore command in order to match how the `/new/import` ux works. 
- install command is not included in the ux flow because it would be "too many options". essentially, probably only a minimal % of users are even configuring this setting so its okay to omit. if it was included more users may accidentally configure it and break their deployments (this parallels to some customer issues we run into along the lines of misunderstanding that they dont _have_ to enter an override when prompted)
-